.github/workflows/examples-nrfconnect.yaml

```diff
@@ -99,7 +99,9 @@ jobs:
               run: scripts/run_in_build_env.sh "python3 scripts/setup/nrfconnect/update_ncs.py --check"
             - name: Run unit tests of factory data generation script
               timeout-minutes: 15
-              run: scripts/run_in_build_env.sh "./scripts/tools/nrfconnect/tests/test_generate_factory_data.py"
+              run: |
+                  scripts/run_in_build_env.sh 'pip3 install -r scripts/setup/requirements.nrfconnect.txt'
```

Comment:
I wonder whether it's worth having this as part of the Docker image...  followup for that if so.